module/applications/actor/base-sheet.mjs

```diff
@@ -695,7 +693,7 @@ export default class ActorSheet5e extends ActorSheet {
     const value = input.value;
     if ( ["+", "-"].includes(value[0]) ) {
       let delta = parseFloat(value);
-      input.value = foundry.utils.getProperty(this.actor, input.name) + delta;
+      input.value = parseFloat(foundry.utils.getProperty(this.actor, input.name)) + delta;
```

Comment:
I think we want to use `Number` here as `Number(null)` is `0`. If we use `parseFloat` for the same we get `NaN`.
```suggestion
      input.value = Number(foundry.utils.getProperty(this.actor, input.name)) + delta;
```